packages/visualizations/addon/templates/components/visualization-config/table.hbs

```diff
@@ -1,4 +1,16 @@
-{{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{#if (check-config 'environment' 'development')}}
```

Comment:
> Can we feature flag this instead?
I do not think feature flag is not proper for this, since it is not a feature. 

> environment doesn't make sense for an addon.

Why `environment ` doesn't make sense?